x-pack/plugins/security_solution/server/types.ts

```diff
@@ -25,6 +26,7 @@ export type SecuritySolutionRequestHandlerContext = RequestHandlerContext & {
   securitySolution: AppRequestContext;
   licensing: LicensingApiRequestHandlerContext;
   alerting: AlertingApiRequestHandlerContext;
+  actions: ActionsApiRequestHandlerContext;
```

Comment:
NOTE: These are "extra's" and not needed for this PR but I figured since I added 'em I should keep 'em since we have them in the plugins and we might use them later on.